config/daemonconfig/daemonconfig.go

```diff
@@ -125,6 +125,9 @@ type DeviceConfig struct {
 // We don't have to persist configuration file for fscache since its configuration
 // is passed through HTTP API.
 func DumpConfigFile(c interface{}, path string) error {
+	if config.IsConfigSourceEnabled() {
+		return nil
```

Comment:
If we don't store this configuration, how can we resolve the situation where the nydus-snapshotter is restarted?

Reply:
Seems with the config `source_enabled`, nydusd does not need config file anymore?

Reply:
After discussing with my supervisor, I will change my plan. The auth configuration will not be stored locally, but will be passed through an API in nydus-snapshotter.